packages/paste-website/src/pages/components/alert/index.mdx

```diff
@@ -91,42 +89,36 @@ Our alerts have icons that both announce the type of alert it is to screen reade
 
 Here are ways that you can compose neutral, warning, error, and dismissible alerts.
 
-<Box marginBottom="space40">
+<Box marginBottom="space30">
   <Alert variant="neutral">
-    <Text as="span">
-      <strong>Martin Luther King, Jr.:</strong> We may have all come on different ships, but we’re in the same boat now.
-    </Text>
+    <strong>Martin Luther King, Jr.:</strong> We may have all come on different ships, but we’re in the same boat now.
   </Alert>
 </Box>
-<Box marginBottom="space40">
+<Box marginBottom="space30">
   <Alert variant="warning">
-    <Text as="span">
-      <strong>James Baldwin:</strong> Love takes off masks that we fear we cannot live without and know we cannot live
-      within.
-    </Text>
+    <strong>James Baldwin:</strong> Love takes off masks that we fear we cannot live without and know we cannot live
+    within.
   </Alert>
 </Box>
-<Box marginBottom="space40">
+<Box marginBottom="space30">
   <Alert variant="error">
-    <Text as="span">
-      <strong>Audre Lorde:</strong> It is not our differences that divide us. It is our inability to recognize, accept,
-      and celebrate those differences.
-    </Text>
+    <strong>Audre Lorde:</strong> It is not our differences that divide us. It is our inability to recognize, accept,
+    and celebrate those differences.
   </Alert>
 </Box>
 <Box marginBottom="space70">
   <Alert onDismiss={() => alert('dismissed')} variant="warning">
-    <Text as="span">
-      <strong>Toni Morrison:</strong> You wanna fly, you got to give up the shit that weighs you down.
-    </Text>
+    <strong>Toni Morrison:</strong> You wanna fly, you got to give up the shit that weighs you down.
   </Alert>
 </Box>
 
 ### Neutral
 
 Use a neutral alert to notify a user of a change that's relevant for them to achieve their current goal. Don't use an alert to communicate general promotional information.
 
-In most cases, a neutral alert should be dismissible (_i.e._, have a close button) and should be assigned `role="status"`
+In most cases, a neutral alert should be dismissible (_i.e._, have a close button) and should be assigned `role="status"`.
+
+You may use [the Text primitive](/primitives/text) as a child of Alert, but if you do, be sure to pass `color="colorTextNeutral"` for neutral alerts.
```

Comment:
suggestion: if you wanted a simpler, more generic tip, you can tell people to use `color="inherit'` for all of these in alert and toast.

Reply:
ah much better